arbitrum-docs/partials/_contribute-docs-partial.md

```diff
@@ -0,0 +1,258 @@
+
+import PublicPreviewBannerPartial from '../partials/_public-preview-banner-partial.md'; 
+
+:::tip Submitting a docs PR?
+
+**Please review this guidance before issuing pull requests** into the [Arbitrum docs repository](https://developer.arbitrum.io/). This guidance helps us **streamline** and **scale** our efforts. Thank you!
+
+:::
+
+<PublicPreviewBannerPartial />
+
+The [`developer.arbitrum.io`](https://developer.arbitrum.io/) docs portal is the **single source of truth** for documentation that supports Offchain's product portfolio. This includes documentation for:
+
+1. [Building dApps](/quickstart-solidity-hardhat) with Arbitrum’s chains.
+2. [Bridging tokens](../getting-started-users.mdx) to Arbitrum’s chains.
+3. [Running an Arbitrum node](../node-running/quickstart-running-a-node.md).
+4. [Launching a self-managed chain](../launch-orbit-chain/orbit-quickstart.md) using Arbitrum Orbit.
+5. [Educational materials](../intro) that explain how these technologies work.
+
+Contributions to each of these content areas are welcome from the entire Ethereum community.
+
+This document provides an overview of the **protocols** and **content conventions** that we use to craft, organize, and publish documentation. Familiarity with [Markdown](https://www.markdownguide.org/basic-syntax/) syntax, Github, and [Docusaurus](https://docusaurus.io/docs) is expected.
+
+
+## Protocols
+
+Just as Arbitrum and Ethereum nodes follow a peer-to-peer protocol, Arbitrum’s content contributors are encouraged to adopt one of the following content collaboration protocols:
+
+### "Core docs" protocol
+
+**Core docs** are documents that help readers use Arbitrum products directly. If a document isn't in a `Third-party content` folder in the sidebar, it's a core document.
+
+Although Offchain Labs is responsible for producing and maintaining core docs, contributions are welcome from all. To contribute a new core doc:
```

Comment:
We specify here how to contribute with a PR, but we should maybe include instructions to contribute via Issues, in case someone sees something that should be fixed (typos, outdated information, broken functionality, ...) but don't have the knowledge to create a PR to fix it.